scripts/release.sh

```diff
@@ -65,8 +65,8 @@ HASH=`get_hash`
 BRANCH=`git rev-parse --abbrev-ref HEAD`
 BRANCH=${BRANCH////-}
 IMAGES=
-DEFAULT_IMAGES="exporters.metrics.pool operators.upgrade obs.callhome"
-IMAGES_THAT_REQUIRE_HELM_CHART=("operators.upgrade")
+DEFAULT_IMAGES="exporters.metrics.pool operators.upgrade obs.callhome upgrade.job"
+IMAGES_THAT_REQUIRE_HELM_CHART=("operators.upgrade" "upgrade.job")
```

Comment:
these 2 upgrades are only here until we replace the old ugrade right?

Reply:
correct